templates/workspace_services/azureml/user_resources/aml_compute/terraform/deploy.sh

```diff
@@ -1,6 +1,14 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
 export TF_LOG=""
+
+# This variables are loaded in for us
+# shellcheck disable=SC2154
 terraform init -input=false -backend=true -reconfigure \
-    -backend-config="resource_group_name=$TF_VAR_mgmt_resource_group_name" \
+    -backend-config="resource_group_name=${TF_VAR_mgmt_resource_group_name}" \
```

Comment:
just curious: why does this line get the `{` brackets added, but others in this command (and other examples in this PR) not?